File: app/src/main/java/com/akribase/archycards/wheelview/transformer/WheelItemTransformer.java
package com.akribase.archycards.wheelview.transformer;

import android.graphics.Rect;

import com.akribase.archycards.wheelview.WheelView;

public interface WheelItemTransformer {
    /**
     * You have control over the Items draw bounds. By supplying your own WheelItemTransformer
     * you must call set bounds on the itemBounds.
     */
    void transform(WheelView.ItemState itemState, Rect itemBounds);
}

File: app/src/main/java/com/akribase/archycards/wheelview/transformer/WheelItemTransformer.kt
package com.akribase.archycards.wheelview.transformer

import android.graphics.Rect
import com.akribase.archycards.wheelview.WheelView.ItemState

interface WheelItemTransformer {
    /**
     * You have control over the Items draw bounds. By supplying your own WheelItemTransformer
     * you must call set bounds on the itemBounds.
     */
    fun transform(itemState: ItemState, itemBounds: Rect)
}